src/lsptoolshost/services/projectContextService.ts

```diff
@@ -22,13 +22,18 @@ export interface ProjectContextChangeEvent {
 export class ProjectContextService {
     private readonly _contextChangeEmitter = new vscode.EventEmitter<ProjectContextChangeEvent>();
     private _source = new vscode.CancellationTokenSource();
+    private readonly _emptyProjectContext: VSProjectContext = {
+        _vs_id: '',
+        _vs_kind: '',
+        _vs_label: '',
```

Comment:
are the empty strings needed for anything?  Might prefer to send undefined if not

Reply:
I do use it to blank out the Project Context item text. Going to leave as is for now and we can revisit if necessary.